setup.py

```diff
@@ -42,8 +42,8 @@
         "python-stdnum >= 1.16, < 2.0.0",
         "pantomime >= 0.4.0, < 1.0.0",
         "pytz >= 2021.1",
-        "rdflib >=5.0.0, < 5.1.0",
-        "networkx >= 2.5, < 2.7",
+        "rdflib >= 5.0.0, < 6.1.0",
```

Comment:
version 6.x introduces breaking changes. Current changes will fail on version 5.x for example. So we should narrow the range down to only version 6.x.

Reply:
Good point. This reflects the dependabot change but I agree it makes no sense across a major version boundary

Reply:
FYI, your merge conflict fix removed the 5.x to 6.x change you made.